client/components/ScenariosList/DeletedCard.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { Button, Card } from 'semantic-ui-react';
+import PropTypes from 'prop-types';
+
+const strike = { textDecoration: 'line-through' };
+
+const DeletedCard = ({ id, title, description }) => {
+    return (
+        <Card className="scenario__entry" key={id}>
+            <Card.Content>
+                <Card.Header style={strike}>{title}</Card.Header>
+                <Card.Description style={strike}>
+                    {description}
+                </Card.Description>
+            </Card.Content>
+            <Card.Content extra>
+                <Button.Group className="scenario__entry--edit-buttons">
+                    <Button>Restore</Button>
+                </Button.Group>
+            </Card.Content>
+        </Card>
+    );
+};
+
+DeletedCard.propTypes = {
+    id: PropTypes.string.required,
```

Comment:
This should be 

```
    id: PropTypes.oneOfType([
        PropTypes.number,
        PropTypes.string
    ]).isRequired,
```

Otherwise the following warning occurs: 

```
Warning: Failed prop type: Invalid prop `id` of type `number` supplied to `DeletedCard`, expected `string`.
```

Reply:
I'm switching it to a `node` since it really could be either a string or a number